dev/com.ibm.ws.org.eclipse.jdt.core/bnd.bnd

```diff
@@ -12,8 +12,8 @@
 bVersion=1.0
 
 Bundle-Name: JDT Compiler
-Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core.3.22.0
-Bundle-Description: JDT Compiler, version 3.22.0 [4.16]
+Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core
+Bundle-Description: JDT Compiler, version 3.22.0: 2020-05-30
```

Comment:
I just started a build, but I think it be be better to change the description to `Eclipse Java Compiler (ECJ)` just to be a bit clearer.

Reply:
2020-05-06 is wrong here.  The release is really 2021-06

Reply:
I took the date from the manifest: `Bundle-Version: 3.22.0.v20200530-2032`

Ah, it was likely built on May 30th, but not released until June, 2020.   This is 3.22, I believe you meant 202**0**-06?

3.26.0 was released 202**1**-06.

Reply:
I'm not sure when the plan is for 3.26 right now. Let me find out.